Documentation/ceph-quickstart.md

```diff
@@ -86,6 +86,35 @@ Before you start the operator in production, there are some settings that you ma
    2. Flex driver: The flex driver is deprecated in favor of the CSI driver, but can still be enabled with the `ROOK_ENABLE_FLEX_DRIVER` setting.
    3. Node affinity and tolerations: The CSI driver by default will run on any node in the cluster. To configure the CSI driver affinity, several settings are available.
 
+## Deploying to other namespaces
+
+If you wish to deploy the Rook Operator and/or Ceph clusters to alternate namespaces, the manifests
+are commented to allow for easy `sed` replacements. Change `ROOK_CLUSTER_NAMESPACE` to tailor the
+manifests for an additional Ceph cluster. You can choose to also change `ROOK_OPERATOR_NAMESPACE` to
+create a new Rook Operator for each Ceph cluster, or you can leave it at the same value for every
+Ceph cluster if you only wish to have one Operator manage all Ceph clusters.
+
+This will help you manage namespaces more easily, but you should still make sure the resources are
+configured to your liking.
+
+```sh
+cd cluster/examples/kubernetes/ceph
+
+export ROOK_OPERATOR_NAMESPACE="rook-ceph"
+export ROOK_CLUSTER_NAMESPACE="rook-ceph"
+
+sed -i.bak \
+    -e "s/\(.*\):.*# namespace:operator/\1: $ROOK_OPERATOR_NAMESPACE # namespace:operator/g" \
+    -e "s/\(.*\):.*# namespace:cluster/\1: $ROOK_CLUSTER_NAMESPACE # namespace:cluster/g" \
+    -e "s/\(.*serviceaccount\):.*:\(.*\) # serviceaccount:namespace:operator/\1:$ROOK_OPERATOR_NAMESPACE:\2 # serviceaccount:namespace:operator/g" \
+    -e "s/\(.*serviceaccount\):.*:\(.*\) # serviceaccount:namespace:cluster/\1:$ROOK_CLUSTER_NAMESPACE:\2 # serviceaccount:namespace:cluster/g" \
+    -e "s/\(.*\): [-_A-Za-z0-9]*\.\(.*\) # driver:namespace:operator/\1: $ROOK_OPERATOR_NAMESPACE.\2 # driver:namespace:operator/g" \
+    -e "s/\(.*\): [-_A-Za-z0-9]*\.\(.*\) # driver:namespace:cluster/\1: $ROOK_CLUSTER_NAMESPACE.\2 # driver:namespace:cluster/g" \
+  common.yaml operator.yaml cluster.yaml # add other files or change these as desired for your config
+
+kubectl apply -f common.yaml -f operator.yaml -f cluster.yaml # add other files as desired for your config
+```
+
```

Comment:
What do you think of this @travisn ?

Reply:
Per separate discussion, I'd recommend this move to the advanced config help doc.